tests/test_floss.py

```diff
@@ -164,6 +170,8 @@ def test_floss():
         D = naive_distance_profile(left_T[-m:], left_T, m)
         D[zone_start:] = np.inf
 
+        left_T_isfinite = np.all(np.isfinite(left_T))
```

Comment:
I'm going to ask you to leave the `test_floss` alone (or revert it back to the original test) and create a separate `test_floss_nan_inf` test. So, `test_floss` tests the best case scenario where the input data does not contain `np.nan`/`np.inf` and `test_floss_nan_inf` is a variation of `test_floss` where the inputs contain `np.nan` and/or `np.inf`. For sanity, we'd want to see that the original unaltered `test_floss` actually passes

Reply:
I believe that this line should be:

```
left_T_isfinite = np.isfinite(left_T)
left_T_subeq_isfinite = np.all(core.rolling(left_T_isfinite, m), axis=1)
D[~left_T_subseq_isfinite] = np.inf
```